packages/ui/src/i18n/dictionaries/authenticator/defaultTexts.ts

```diff
@@ -58,8 +62,11 @@ export const defaultTexts = {
   RESEND_CODE: 'Resend Code',
   RESET_PASSWORD_HEADING: 'Reset your password',
   RESET_PASSWORD: 'Reset Password',
+  SELECT_MFA_TYPE_LABEL: 'Select MFA Type',
+  SELECT_MFA_TYPE_PLACEHOLDER: 'Please select desired MFA type',
```

Comment:
Would drop the _LABEL and _PLACEHOLDER suffixes since that pattern does not exist in the `Authenticator` text libs